### DISCUSSION TITLE
Deployment examples

### DISCUSSION THREAD

**Discussion by fforootd** [zitadel/zitadel#7332] [Polls]
Have you ever wondered where or how you should deploy ZITADEL?
At least I have... so I thought lets create a poll!

Lets vote and we/I will make examples for the most popular platforms 😁

**Comment by bddvlpr** [comment]
NixOS would be great :)

**Reply by fforootd** [reply]
Great pick ~~let me add that to the list 😁~~ (no edits allowed it seems)

**Comment by benperiton** [comment]
Nomad

**Reply by fforootd** [reply]
Nice pick as well!

**Reply by sevensolutions** [reply]
Iam also interested in running ZITADEL on Nomad. Does anyone have a job sample including config for rolling upgrades?

**Comment by samip5** [comment]
systemd-user units, without container of any sort meaning running the app directly on host.

**Reply by fforootd** [reply]
🚀

**Reply by craiglawson** [reply]
+1 on this

**Comment by jobcespedes** [comment]
K8s

**Reply by fforootd** [reply]
OK, have you seen this https://github.com/zitadel/zitadel-charts/tree/main/examples is there something missing?

**Reply by jobcespedes** [reply]
Thanks. First time taking a look at it. At a glance, nothing is missing.

**Reply by fforootd** [reply]
Good to hear!

**Comment by Trickfilm400** [comment]
Some niche / edge cases would be a Plesk hosting (with MySQL and without docker, as it seems not supported by plesk...)

But if this is technically not possible due to the architecture its not an issue

**Reply by fforootd** [reply]
Not sure how plesk work (too long not touched it) 😁 

Supporting different databases is not a high priority right now #3598

**Comment by edr3x** [comment]
Primarily on K8S and sometimes on Docker with Compose

**Reply by fforootd** [reply]
Great, have you seen these guides https://zitadel.com/docs/self-hosting/deploy/compose and https://github.com/zitadel/zitadel-charts/tree/main/examples

**Comment by compgeniuses** [comment]
coolify.io would be wonderful to have install for.

**Reply by fforootd** [reply]
Nice choice

**Comment by compgeniuses** [comment]
i am working on a NEthserver 8 module for zitadel

**Reply by fforootd** [reply]
Oh cool, I have not known this project!

**Reply by compgeniuses** [reply]
here is the link to the Application website: https://www.nethserver.org/nethserver-8/

[NEthserver 8](https://github.com/NethServer/ns8-core) IS a Re-workd of NEthserver 7, which is a Linux Small Business Server, that offers Active directory, Fie Server MAil and other Modules into one.

I Would be happy to work with you guys on an Nethserver 8 Module for Zitadel that is officially Supported by you guys. Would be Awesome.

You begin by Using this kickstart template https://github.com/NethServer/ns8-kickstart. then you add All you need the Module to Do, Similar to the other Modules.

IF you have questions, there is an active community here: https://community.nethserver.org

**Reply by compgeniuses** [reply]
HEllo, i never posted an Update. Zitadel for NEthserver is now on BETA for testing, and can directly intergrate with the provided Ldap on Nethserver here is the App https://github.com/geniusdynamics/ns8-zitadel

And was Announced on the community here: https://community.nethserver.org/t/zitadel-sso-multi-tenant-identity-infrastructure/24057

**Comment by muhlemmer** [comment]
Simple (hetzner) VM with Linux and docker compose.

**Reply by tafaust** [reply]
I second Hetzner.

**Comment by danfarewell** [comment]
Coolify! :)

**Reply by fforootd** [reply]
Just for reference, even though I know it is by you https://github.com/zitadel/zitadel/discussions/8370

**Reply by sgac-ops** [reply]
Did you get it working on Coolify?

**Comment by dkdndes** [comment]
Open Source Video Conference https://jitsi.org would be nice.
